greymatter/modules/fabric/services.tf

```diff
@@ -1,3 +1,42 @@
+# service discovery
+
+resource "aws_service_discovery_private_dns_namespace" "greymatter" {
+  name        = var.dns_ns_name
+  description = "Service discovery namespace for Grey Matter Fabric"
+  vpc = var.vpc_id
+}
+
+resource "aws_service_discovery_service" "control-api" {
```

Comment:
In your tests, what did the route53 record actually look like when this was generated?

Reply:
<img width="985" alt="Screen Shot 2020-09-02 at 12 58 37 PM" src="https://user-images.githubusercontent.com/32151242/92013624-0c553780-ed1c-11ea-9b59-521b03f24d92.png">

Reply:
@joshua-rutherford this in particular you should check out. It's not a "normal" route53 resource that we declare, but it shows up as regular DNS records.